python/cuml/tests/test_stratified_kfold.py

```diff
@@ -63,3 +65,53 @@ def test_invalid_folds(n_splits):
         kf = StratifiedKFold(n_splits=n_splits)
         for train_index, test_index in kf.split(X, y):
             break
+
+
+@pytest.mark.parametrize("shuffle", [True, False])
+@pytest.mark.parametrize("n_splits", [5, 10])
+@pytest.mark.parametrize(
+    "rs",
+    [
+        1,
+        np.random.RandomState(1),
+        cp.random.RandomState(1),
+        np.random.default_rng(1),
+        cp.random.default_rng(1),
+    ],
+)
+def test_kfold(shuffle: bool, n_splits: int, rs) -> None:
```

Comment:
```suggestion
    "random_state",
    [
        1,
        np.random.RandomState(1),
        cp.random.RandomState(1),
        np.random.default_rng(1),
        cp.random.default_rng(1),
    ],
)
def test_kfold(shuffle, n_splits, random_state) -> None:
```

I'd spell out random state to make it easier to read, remove the type annotations